tools/scaling/sweep/node_to_node/unstructured_mesh/strong/strong_scaling.py

```diff
@@ -19,9 +19,10 @@
     from pyopensn.mat import SetProperty, ISOTROPIC_MG_SOURCE, FROM_ARRAY
     from pyopensn import mat, materials
 
-Ng = 64          # Number of energy groups
-Npolar = 14      # Number of polar angles
-Nazimuthal = 32  # Number of azimuthal angles
+Ng = 64              # Number of energy groups
+Npolar = 14          # Number of polar angles
+Nazimuthal = 32      # Number of azimuthal angles
+ScatteringOrder = 0  # Scattering order
```

Comment:
it would be probably better to use snake_case for the variables here and in the other script too.

I think this `N` prefix is some old leftover from chi-tech (maybe).
Also, when I saw `ScatteringOrder` below, I thought that was a class.

Reply:
Fixed.